package.json

```diff
@@ -147,7 +149,7 @@
     "dev": "yarn prebuild && next dev",
     "spellcheck": "cspell \"src/**/*.mdx\" --no-progress",
     "spellcheck-diff": "git diff --name-only --cached | awk \"/src.*\\.mdx/{print}\" | npx cspell --no-must-find-files --file-list stdin",
-    "build": "node tasks/generate-sitemap.mjs && next build && next export -o client/www/next-build",
+    "build": "node tasks/generate-sitemap.mjs && yarn playwright install && next build && next export -o client/www/next-build",
```

Comment:
Is there a reason why we don't just install playwright as a dev dependency?

Reply:
It is installed as a dev dependency but it needs to install the browsers on the build.